imgui_demo.cpp

```diff
@@ -7465,8 +7465,8 @@ static void ShowExampleAppFullscreen(bool* p_open)
 // [SECTION] Example App: Manipulating Window Titles / ShowExampleAppWindowTitles()
 //-----------------------------------------------------------------------------
 
-// Demonstrate using "##" and "###" in identifiers to manipulate ID generation.
```

Comment:
I'm not sure there is much point making this comment on a closed pull request, but given the effort put in here, this line should be "Demonstrate use of" not "Demonstrate using of"

Reply:
Fixed in #5681, thanks for the review.